src/mixer/basetrackplayer.h

```diff
@@ -41,6 +42,8 @@ class BaseTrackPlayer : public BasePlayer {
     void loadTrackFailed(TrackPointer pTrack);
     void newTrackLoaded(TrackPointer pLoadedTrack);
     void unloadingTrack(TrackPointer pAboutToBeUnloaded);
+    void noPassthroughInputConfigured();
+    void noVinylControlInputConfigured();
 };
 
 class BaseTrackPlayerImpl : public BaseTrackPlayer {
```

Comment:
I was just confused again when I read this PR. 
With a good refactoring tool the change is an issue of a few minutes. 
But I agree, that there is no strong need to change it now.